### DISCUSSION TITLE
Feedback: FAQ

### DISCUSSION THREAD

**Discussion by ewaldtlgs803** [mswjs/mswjs.io#320] [Q&A]
- [Open page](https://mswjs.io/docs/faq/)

## Feedback
After upgrading to 2.0, I had the TextEncoder issue and followed the instructions to add the polyfill, but unfortunately it did not resolve the issue. Both the jest.polyfills.js and the jest.config.js are in the root of my react app.

**Comment by kettanaito** [comment]
Hey, @ewaldtlgs803. Can you share a reproduction repository for this? 

Also, please take a look at our [Jest + MSW integration example](https://github.com/mswjs/examples/tree/main/examples/with-jest). It features the polyfill and all the configurations you need to make Jest work. I trust you will spot a difference with your setup and that will be it.
